source/views/streaming/streams/list.js

```diff
@@ -74,31 +73,26 @@ export class StreamListView extends React.PureComponent<Props, State> {
 			const dateFrom = date.format('YYYY-MM-DD')
 			const dateTo = date
 				.clone()
-				.add(1, 'month')
+				.add(2, 'month')
 				.format('YYYY-MM-DD')
 
 			let params = {
-				class: 'current',
 				sort: 'ascending',
 				// eslint-disable-next-line camelcase
 				date_from: dateFrom,
 				// eslint-disable-next-line camelcase
 				date_to: dateTo,
 			}
 
-			const streamsAPI = `${url}?${qs.stringify(params)}`
-			const data = await fetchJson(streamsAPI)
-			const streams = data.results
+			const data = await fetchJson(API('/streams/upcoming', params))
 
 			// force title-case on the stream types, to prevent not-actually-duplicate headings
-			const processed = streams
+			const processed = data
 				.filter(stream => stream.category !== 'athletics')
 				.map(stream => {
 					const date = moment(stream.starttime, 'YYYY-MM-DD HH:mm')
 					const group =
-						stream.status.toLowerCase() !== 'live'
-							? date.format('dddd, MMMM Do')
-							: 'Live'
+						stream.status !== 'live' ? date.format('dddd, MMMM Do') : 'Live'
```

Comment:
keep the toLowerCase

Reply:
There's no reason to if the API sticks to the spec of lowercase

Reply:
But we'll do it just to be safe.

Reply:
You're right. We should not do this in the app. This sort of consistency check is what the server is for. 

You can remove the toLlwerCase again… sorry.